<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GAME</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/pics/lizard.ico</normaloff>:/pics/lizard.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>70</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>181</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>181</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 0);
background-image: url(:/pics/spock.png);
border-radius:10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>70</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>181</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>181</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pics/lizard.png);
border-radius:10px;
background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>338</x>
       <y>70</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>181</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>181</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-image: url(:/pics/scissors.png);
background-color: rgb(255, 255, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>70</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>181</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>181</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
background-image: url(:/pics/paper.png);

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>641</x>
       <y>70</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>181</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>181</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(170, 170, 255);
background-image: url(:/pics/rock.png);


</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="spock_btn">
     <property name="geometry">
      <rect>
       <x>37</x>
       <y>260</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(35,35,35);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(200,200,200);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:15px;
}
QPushButton:pressed {	
	background-color: rgb(250,0,0);
	border:2px solid rgb(201,52,19);
	color: rgb(250,250,250);
	border-radius:20px;
}</string>
     </property>
     <property name="text">
      <string>SPOCK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lizard_btn">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>260</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(35,35,35);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(200,200,200);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:15px;
}
QPushButton:pressed {	
	background-color: rgb(250,0,0);
	border:2px solid rgb(201,52,19);
	color: rgb(250,250,250);
	border-radius:20px;
}</string>
     </property>
     <property name="text">
      <string>LIZARD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scissors_btn">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>260</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(35,35,35);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(200,200,200);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:15px;
}
QPushButton:pressed {	
	background-color: rgb(250,0,0);
	border:2px solid rgb(201,52,19);
	color: rgb(250,250,250);
	border-radius:20px;
}</string>
     </property>
     <property name="text">
      <string>SCISSORS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paper_btn">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>260</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(35,35,35);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(200,200,200);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:15px;
}
QPushButton:pressed {	
	background-color: rgb(250,0,0);
	border:2px solid rgb(201,52,19);
	color: rgb(250,250,250);
	border-radius:20px;
}</string>
     </property>
     <property name="text">
      <string>PAPER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rock_btn">
     <property name="geometry">
      <rect>
       <x>651</x>
       <y>260</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(35,35,35);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(200,200,200);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:15px;
}
QPushButton:pressed {	
	background-color: rgb(250,0,0);
	border:2px solid rgb(201,52,19);
	color: rgb(250,250,250);
	border-radius:20px;
}</string>
     </property>
     <property name="text">
      <string>ROCK</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>310</y>
       <width>744</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(91, 91, 91);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="user_pic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>131</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(170, 170, 127);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pc_pic">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>131</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(170, 170, 127);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 127);
border:2px solid rgb(60,60,60);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>YOU</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 127);
border:2px solid rgb(60,60,60);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>COMPUTER</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>166</x>
        <y>130</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>VS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="result_label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>166</y>
        <width>361</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 127);
background-color: rgb(91, 91, 91);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="phrase_label">
      <property name="geometry">
       <rect>
        <x>379</x>
        <y>102</y>
        <width>361</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 170, 127);
background-color: rgb(91, 91, 91);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>409</x>
        <y>10</y>
        <width>321</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
background-color: rgb(80, 80, 80);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="score_user">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>22</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0,0,0);
border:4px solid rgb(30,30,30);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="score_pc">
       <property name="geometry">
        <rect>
         <x>174</x>
         <y>20</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
border:4px solid rgb(30,30,30);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SCORE BOARD</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>74</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>YOU</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>191</x>
         <y>72</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>COMPUTER</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
     <zorder>label_7</zorder>
     <zorder>user_pic</zorder>
     <zorder>label_8</zorder>
     <zorder>pc_pic</zorder>
     <zorder>label_9</zorder>
     <zorder>result_label</zorder>
     <zorder>phrase_label</zorder>
     <zorder>frame_3</zorder>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>5</y>
       <width>641</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="text">
      <string>CHOOSE WISELY!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="rules_btn">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(105,105,105);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(200,200,200);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:13px;
}
QPushButton:pressed {	
	background-color: rgb(250,0,0);
	border:2px solid rgb(201,52,19);
	color: rgb(250,250,250);
	border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>RULES</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
